---
title: "About"
subtitle: "I am a Scientific Investigator for Computational Biology at [GlaxoSmithKline](https://www.gsk.com/en-gb/)"
title-block-banner: true
toc: true
about:
  template: solana
  links:
    # - icon: linkedin
    #   text: LinkedIn
    #   href: https://www.linkedin.com/in/ashwinkr/
    #   target: _blank
    # - icon: github
    #   text: Github
    #   href: https://github.com/ashwini-kr-sharma/
    #   target: _blank
    - text: <i class="ai ai-google-scholar" role="img"></i> Google Scholar
      href: https://goo.gl/tvIoC2
      target: _blank
    - text: <i class="ai ai-orcid" role="img"></i> Orchid
      href: https://goo.gl/YW1Kcq
      target: _blank
---

<!-- {{< ai google-scholar >}} -->
---

### Management Skills

- **Cross-functional team leadership:** I have worked in Matrix teams across multiple projects as the Computational Biology lead and  provided overall co-leadership to interdisciplinary projects. Communicated key insights to senior leadership to impact decision-making on early target portfolios

- **Strategic planning & Project Management:** Set clear objectives and deliverables. Planned & executed computational biology experiments spread over > 1 year timescale. Managed talent resourcing, resolved blockers and met delivery timelines.

- **Scientific communication:** All my projects have involved effectively communicating Computational Biology derived insights to leadership to influence decision making

---

### Technical Skills

- **Single cell analysis:** Expertise in single-cell RNAseq & ATACseq data analysis. Have performed data analyses for data QC, cell type annotation, data integration, trajectory inference, cell marker identification, differential gene expression analysis, transcriptional activity & gene regulation analysis

- **Genome scale pooled & Arrayed Perturbation screening data analysis:** Expertise in high-throughput screening technologies like PerturbSeq/CROPseq, DRUGseq, PARSE, 10x & CRISPR-FACS-based functional genomics screens. Have performed data analyses for Hit identification, Target mechanism of action evaluation, Perturbation signature scoring etc

- **Analysis of data generated from large public genomic and epigenomic datasets:** Expertise in analyzing data from RNAseq, CHIPseq, ATACseq, HiC, Methylation array, Microarray etc. Have worked extensively with data generated by consortium like TCGA, GTeX, ENCODE, BLUEPRINT, DepMap etc

- **Multi-omic data interpretation & integration:** Expertise in analysing data generated from various NGS technologies - Data clustering, Pattern identification, Differential expression/activity/abundance analysis, Pathway analysis, Regulatory network inference, Multi-modal data integration, Biomarker discovery, Scoring molecular signatures, Patient stratification, Associating signatures to clinical outcomes, Data visualization etc

- **Disease Genotype-to-Phenotype association studies:** Have expertise in integrating genetics (GWAS) and (epi-)genomics data using tools like sLDSC, MAGMA & scLinker to infer disease associated mechanisms

- **Machine Learning:** Expertise in applying supervised and unsupervised machine learning to NGS datasets - Non-negative matrix factorization, data classification & feature selection etc

- **Reproducible research:** Have experience in Git, Github, Quarto, Rmarkdown, Plotly, Shiny, Nextflow, Snakemake & Docker

- **Coding/Tools:** Have experience in R statistical programming, Bioconductor, Python, Bash & Conda

---

### Academic background

#### Postdoctoral Research
::: {style="font-size: 75%;"}
(*2016-2021*)
:::

::: {style="font-size: 90%;"}
> - [Health Data Science Unit](https://www.hdsu.org/), Universitätsklinikum Heidelberg, Germany 
> - [Institute of Pharmacy and Molecular Biotechnology and BioQuant](https://www.bioquant.uni-heidelberg.de/groups/herrmann), University Heidelberg, Germany
> - [Division of Neuroblastoma Genomics](https://www.dkfz.de/en/neuroblastom-genomik/index.php), German Cancer Research Center (DKFZ), Heidelberg, Germany
:::

My postdoctoral research focused on identifying the enhancer-driven [regulatory subtypes in Neuroblastoma](https://doi.org/10.1038/s43018-020-00145-w) and understanding its [developmental origins](https://doi.org/10.1038/s41588-021-00806-1)

#### Doctoral Research (PhD)
::: {style="font-size: 75%;"}
 (*2011-2016*)
:::

::: {style="font-size: 90%;"}
> - [German Cancer Research Center (DKFZ)](https://www.dkfz.de/en/index.html), Heidelberg, Germany
> - [Hans Knöll Institute (HKI)](http://www.leibniz-hki.de/en/home.html), Jena, Germany
:::

My [doctoral research](10.11588/heidok.00020332) was on understanding the reprogramming of the [metabolic landscape in tumors](https://doi.org/10.1158/0008-5472.CAN-15-235)

I was also involved in 5 different collaborations with experimental groups as lead computational biologist in topics related to non-coding RNA, immunotherapy, biomarker discovery and mechanism of action of a chemotherapeutic drug

#### MSc Genomics
::: {style="font-size: 75%;"}
(*2009-2011*)
:::

::: {style="font-size: 90%;"}
> - [Madurai Kamraj University](http://www.genomicsmku.org/), Madurai, India
:::

For my Master's thesis, I worked on the structure-based rational design of a peptide inhibitor against HIF1alpha-HRE binding and its structural studies. I performed computational protein structure modeling and molecular dynamics studies
                  
For my internship project, I worked on the expression, purification, crystallization and *in-silico* modeling of the *FadD9* protein from Mycobacterium tuberculosis. I performed cloning & transformation assays, protein isolation, western blotting, protein purification using chromatography techniques, protein crystallization and computational protein structure modeling

#### BSc Biotechnology
::: {style="font-size: 75%;"}
(*2006-2009*)
:::

::: {style="font-size: 90%;"}
> - [Vellore Institute of Technology](http://www.vit.ac.in/), Vellore, India.
:::
