---
title: "About"
title-block-banner: true
---

I am a Scientific Investigator - Computational Biology at [GlaxoSmithKline (GSK)](https://www.gsk.com/en-gb/). 

### Academic background

As a postdoc, I worked at the [Health Data Science Unit](https://www.hdsu.org/), Universitätsklinikum Heidelberg, Germany. My research focused on identifying the enhancer-driven regulatory subtypes in Neuroblastoma and understanding its developmental origins.

I did my doctoral research at the [German Cancer Research Center (DKFZ)](https://www.dkfz.de/en/index.html) and [Hans Knöll Institute (HKI)](http://www.leibniz-hki.de/en/home.html). My research focussed on the reprogramming of the metabolic landscape in tumors.

I have a Masters degree in Genomics from the [Madurai Kamraj University](http://www.genomicsmku.org/), India and a Bachelors degree in Biotechnology from [Vellore Institute of Technology](http://www.vit.ac.in/), India.

---

### Management Skills

- **Cross-functional team leadership:** I have worked in Matrix teams across multiple projects as the Computational Biology project lead
  - **Strategic planning & Project Management:** I have contributed to wet-lab experiment design involving downstream bioinformatics analysis, planned for Computational Biology experiments, talent resourcing, and timelines across projects
- **Stakeholder management:** I have communicated key high-level insights to senior leaders and decision makers to impact broader company vision and direction.
- **Scientific communication:** All my projects have involved communicating Computational Biology derived insights to non-experts in interdisciplinary projects to impact decision making


### Technical Skills

- **Next generation sequencing data analysis:**  Data generated with genomic and epi-genomic technologies like: scRNAseq, RNAseq, CHIPseq, ATACseq, HiC, Micro-array, 450/850K methylation array etc
- **Genome scale and Arrayed Perturbation screening data analysis** - Hit Identification, Mechanism of action, Perturbation signature scoring etc
- **Multi -omic data integration and interpretation** - Biological pathway analysis, Regulatory networks, Biomarker discovery, scoring molecular signatures in disease cohorts and associating it to clinical outcomes, Data visualization etc 
- **Single cell transcriptomics analysis** - QC, cell type and cell state identification, cell marker identification, transcriptional activity etc
- **Disease heritability and GWAS enrichment** - Disease - Mechanism association analyses using statistical genetics methods like sLDSC, MAGMA, scLinker etc
- **Analyses of large public -omic datasets** - generated by consortium like TCGA, GTeX, ENCODE, BLUEPRINT, DepMap etc
- **Machine Learning** - Unsupervised and Supervised learning, Classification and Feature selection (using R packages like caret, glmnet etc)
- **Reproducible research** - [Git/Github](https://github.com/ashwini-kr-sharma/), Rmarkdown, Plotly, ShinyApps, Snakemake, [Docker](https://hub.docker.com/u/ashwinikrsharma)
- **Cloud Computing** - Open Stack, Google Cloud
- **Coding/Tools** - R statistical programming, Bash, Bioconductor, Conda

---

::: {#landing-icons}
[{{< fa brands github size=2x title="github" >}}](https://github.com/ashwini-kr-sharma/){target="_blank"} &nbsp; 
[{{< fa brands google-scholar size=2x title="Google Scholar" >}}](https://goo.gl/tvIoC2){target="_blank"}
:::