---
title: "About"
---

I am a Scientific Investigator - Computational Biology at [GlaxoSmithKline (GSK)](https://www.gsk.com/en-gb/). 

### Academic background

As a postdoc, I worked at the [Health Data Science Unit](https://www.hdsu.org/), Universitätsklinikum Heidelberg, Germany. My research focused on identifying the enhancer-driven regulatory subtypes in Neuroblastoma and understanding its developmental origins.

I did my doctoral research at the [German Cancer Research Center (DKFZ)](https://www.dkfz.de/en/index.html) and [Hans Knöll Institute (HKI)](http://www.leibniz-hki.de/en/home.html). My research focussed on the reprogramming of the metabolic landscape in tumors.

I have a Masters degree in Genomics from the [Madurai Kamraj University](http://www.genomicsmku.org/), India and a Bachelors degree in Biotechnology from [Vellore Institute of Technology](http://www.vit.ac.in/), India.

---

### Technical Skills

- **Next generation sequencing data analysis** - I have extensive experience working with data generated with: scRNAseq, RNAseq, CHIPseq, ATACseq, HiC, Micro-array, 450/850K methylation array etc
- **Genome scale and Arrayed Perturbation screening data analysis** - Hit Identification, Knockout mechanism of action, Perturbation signature scoring etc
- **Disease heritability and GWAS enrichment** - sLDSC, MAGMA, scLinker etc
- **Single cell transcriptomics analysis** - QC, cell type identification, trajectory inference, cell markers, transcriptional activity etc 
- **Analyses of large public -omic datasets** - generated by consortiums like TCGA, GTeX, ENCODE, BLUEPRINT, DepMap etc
- **Multi -omic data integration** - Biological pathway analysis, Regulatory networks, Biomarker discovery, scoring molecular signatures in disease cohorts and associating it to clinical outcomes, Data visualization etc 
- **Machine Learning** - Unsupervised and Supervised learning, Classification and Feature selection (using R packages like caret, glmnet etc)
- **Reproducible research** - Git, Rmarkdown, Plotly, ShinyApps, Snakemake, Docker
- **Cloud Computing** - Open Stack, Google Cloud
- **Coding/Tools** - R statistical programming, Bash, Bioconductor, Conda



- Domain knowledge of tumor biology, molecular biology, metabolism, biochemistry and (epi-)genomics
- Multi -omic data analysis – *RNAseq, Microarray, 450/850K methylation array, CHIPseq, ATACseq* etc
- Experience in analysing high throughput screening datasets
- Experience with single-cell technologies, especially *scRNA-seq*
- Machine Learning with focus on classification, regression and feature selection problems
- Experience in handling data from *TCGA, GTeX, ENCODE, DepMap, Human cell atlas* etc
- Scoring of molecular phenotypes using established gene signatures
- Identification of molecular signatures in disease cohorts and associating it to clinical outcomes
- Running and maintaining established NGS pipelines
- Biological pathway analysis, Biomarker discovery, Pattern recognition, Data visualisation etc
- Data interpretation, quality control, result documentation and effective communication
- R statistical programming, Bioconductor, Bash, [Github](https://github.com/ashwini-kr-sharma/), Snakemake, Rmarkdown, Shiny, [Docker](https://hub.docker.com/u/ashwinikrsharma)
- Some experience in cloud computing - OpenStack


### Interpersonal Skills

Most of my projects were carried out in close collaboration with experimental or clinical groups. Thus, I have the necessary experience in managing and working in multidisciplinary teams. Furthermore, my academic background was in experimental biology while my professional background is in computational biology which has made me a good intermediary who can bridge and communicate across diverse experts in interdisciplinary projects.

---


::: {#landing-icons}
[{{< fa brands github size=2x title="github" >}}](https://github.com/ashwini-kr-sharma/){target="_blank"} &nbsp; 
[{{< fa brands google-scholar size=2x title="Google Scholar" >}}](https://goo.gl/tvIoC2){target="_blank"}
:::