---
title: "About"
subtitle: "I am a Scientific Investigator - Computational Biology at [GlaxoSmithKline](https://www.gsk.com/en-gb/)"
title-block-banner: true
---
  
### Academic background

My postdoctoral research focused on identifying the enhancer-driven regulatory subtypes in Neuroblastoma and understanding its developmental origins. I worked at the [Health Data Science Unit](https://www.hdsu.org/), Universitätsklinikum Heidelberg, Germany.

My doctoral research was on understanding the reprogramming of the metabolic landscape in tumors. I worked at the [German Cancer Research Center (DKFZ)](https://www.dkfz.de/en/index.html), Heidelberg and the [Hans Knöll Institute (HKI)](http://www.leibniz-hki.de/en/home.html) in Jena.

I have a Masters degree in Genomics from the [Madurai Kamraj University](http://www.genomicsmku.org/), India and a Bachelors degree in Biotechnology from [Vellore Institute of Technology](http://www.vit.ac.in/), India.

---

### Management Skills

- **Cross-functional team leadership:** I have worked in Matrix teams across multiple projects as the Computational Biology lead and  provided leadership to interdisciplinary projects. Communicated key insights to senior leadership to impact decision-making on early target portfolios

- **Strategic planning & Project Management:** Set clear objectives and deliverables. Planned & executed computational biology experiments spread over 1 year + timescale. Managed talent resourcing, resolved blockers and met delivery timelines.

- **Scientific communication:** All my projects have involved effectively communicating Computational Biology derived insights to leadership to impact decision making

---

### Technical Skills


- **Single cell analysis:** Expertise in single-cell RNAseq & ATACseq data analysis. Have performed data analyses for data QC, cell type annotation, data integration, trajectory inference, cell marker identification, differential gene expression analysis, transcriptional activity & gene regulation analysis etc

- **Genome scale pooled & Arrayed Perturbation screening data analysis:** Expertise in high-throughput screening technologies like PerturbSeq/CROPseq, DRUGseq, PARSE, 10x & CRISPR-FACS-based functional genomics screens. Have performed data analyses for Hit identification, Target mechanism of action evaluation, Perturbation signature scoring etc

- **Analysis of data generated from large public genomic and epigenomic datasets:** Expertise in analyzing data from RNAseq, CHIPseq, ATACseq, HiC, Methylation array, Microarray etc. Have worked extensively with data generated by consortium like TCGA, GTeX, ENCODE, BLUEPRINT, DepMap etc

- **Multi-omic data interpretation & integration:** Expertise in analysing data generated from various NGS technologies - Data clustering, Pattern identification, Differential expression/activity/abundance analysis, Pathway analysis, Regulatory network inference, Multi-modal data integration, Biomarker discovery, Scoring molecular signatures, Patient stratification, Associating signatures to clinical outcomes, Data visualization etc

- **Disease Genotype-to-Phenotype association studies:** Have expertise in integrating genetics (GWAS) and (epi-)genomics data using tools like sLDSC, MAGMA & scLinker to infer disease associated mechanisms

- **Machine Learning:** Expertise in applying supervised and unsupervised machine learning to NGS datasets - Non-negative matrix factorization, data classification & feature selection etc

- **Reproducible research:** Have experience in Git, Github, Quarto, Rmarkdown, Plotly, Shiny, Nextflow, Snakemake & Docker

- **Coding/Tools:** Have experience in R statistical programming, Bioconductor, Python, Bash & Conda

---

::: {#landing-icons}
[{{< fa brands github size=2x title="github" >}}](https://github.com/ashwini-kr-sharma/){target="_blank"} &nbsp; 
[{{< fa brands google-scholar size=2x title="Google Scholar" >}}](https://goo.gl/tvIoC2){target="_blank"}
:::